pom.xml

```diff
@@ -292,7 +292,7 @@ agnostic home for software distribution comprehension and audit tools.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-project-info-reports-plugin</artifactId>
-        <version>3.6.1</version>
```

Comment:
Pls merge the current master into the branch in order to get the same versions as master.